src/css/style.css

```diff
@@ -60,6 +60,140 @@ nav ul li {
   margin: 0 5%;
 }
 
+/* MAIN PART */
+main {
+  background: linear-gradient(-45deg, #081f37, #5fc9f3, #2e79ba, #1e549f);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+}
+
+@keyframes gradient {
+  0% {
+    background-position: 0 50%;
+  }
+
+  50% {
+    background-position: 100% 50%;
+  }
+
+  100% {
+    background-position: 0 50%;
+  }
```

Comment:
The use of a dynamic background is very interesting and give a modern touch to the UI.